website/content/en/preview/contributing/development-guide.md

```diff
@@ -61,6 +61,24 @@ make presubmit # run codegen, lint, and tests
 
 If you are only interested in building the Karpenter images and not deploying the updated release to your cluster immediately with Helm, you can run
 
+*Note: that this will produce a build with the latest version of https://github.com/kubernetes-sigs/karpenter.
+
+```bash
+export $
+make image # build and push the karpenter images
```

Comment:
This is automatically done by `make apply`. Was there a reason you wanted to specifically include this as a separate step ?

Reply:
Same as my comment below. My company is a heavy GitOps shop and I don't deploy karpenter via helm. `make apply` is not relevant to my experience and would actually cause more problems.